tumbleweed

```diff
@@ -161,6 +161,11 @@ tumbleweed_switch()
     echo "choosing latest version"
   fi
 
+  if [ -z "$version" ] ; then
```

Comment:
I believe the preferred way to check empty variable is `[ -z ${version+x}` ]`. Perhaps move this into the `tumbleweed_latest` part of conditional local since the other already performs a check.

Reply:
I see. I will update it.